configure.ac

```diff
@@ -369,7 +369,9 @@ AC_CONFIG_FILES([Makefile
   src/Makefile
   test/Makefile
   test/unit/Makefile
-  test/performance/Makefile])
+  test/performance/Makefile
+  test/performance/shmem_perf_suite/Makefile
+  test/performance/tests/Makefile])
```

Comment:
Could you check the RPM scripts to see if the added directories impact the RPM build?